costmodel/costmodel.go

```diff
@@ -165,7 +165,7 @@ type PrometheusMetadata struct {
 
 // ValidatePrometheus tells the model what data prometheus has on it.
 func ValidatePrometheus(cli prometheusClient.Client) (*PrometheusMetadata, error) {
-	data, err := Query(cli, "up")
+	data, err := Query(cli, "up offset 3h")
```

Comment:
We use this same query to validate the non-thanos Prometheus client as well right? Still ok to offset here?

Reply:
No.. good catch.